src/utils/getPreparedPeople.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable max-len */
+import { Person } from '../types';
+
+export function getPreparedPeople(
+  peopleArray: Person[],
+  { query, sex, centuries }: { query: string; sex: string; centuries: string[] },
+) {
+  let preparedPeople = [...peopleArray];
+
+  if (query) {
+    preparedPeople = preparedPeople.filter((person) => {
+      return (
+        person.name.toLowerCase().includes(query.toLowerCase().trim())
+        || person.motherName?.toLowerCase().includes(query.toLowerCase().trim())
+        || person.fatherName?.toLowerCase().includes(query.toLowerCase().trim())
+      );
+    });
+  }
```

Comment:
it is much better to make variable for normalized query to avoid transforming and triming query for every person for 3 times
line 11: `const normalazidQuery = query.toLowerCase().trim()`
and then use it